packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/index.js

```diff
@@ -63,14 +61,7 @@ const ListView = () => {
     },
   }));
 
-  const handleToggle = (id) => {
-    setIsModalOpen((prev) => !prev);
-
-    if (data.results && id) {
-      const actionData = data.results.find((action) => action.id === id);
-      setDetailsActionData(actionData);
-    }
-  };
+  const [openedLogId, setOpenedLogId] = useState(null);
```

Comment:
In my opinion it's not your best variable name 🤣 , but I don't have much better, here's a suggestion but what you have is fine too

```suggestion
  const [modalLogId, setModalLogId] = useState(null);
```

Reply:
lol you're right